Libraries/Image/NativeImageStore.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+
+export interface Spec extends TurboModule {
+  +hasImageForTag: (
+    uri: string, 
+    callback: (hasImage: boolean) => void,
+  ) => void;
+  +removeImageForTag: (
+    uri: string,
+  ) => void;
+  +addImageFromBase64: (
+    base64ImageData: string,
+    succees: (uri: string) => void,
+    failure: (error: string) => void;
```

Comment:
I don’t think failure callback’s pass back a string arg. I think they are ‘Object’ types.

Reply:
Would it then look something like this?
```
failure: Object;
```
OR
```
failure: (error: Object) => void;
```

Reply:
`failure: (error: Object) => void,`